health/egov-persister/expense-bill-payment-persister.yaml

```
@@ -0,0 +1,413 @@
+       - jsonPath: $.bill.billDetails.*.payee.auditDetails.createdTime
+
+       - jsonPath: $.bill.billDetails.*.payee.auditDetails.lastModifiedBy
+
+       - jsonPath: $.bill.billDetails.*.payee.auditDetails.lastModifiedTime
+        
+       
+    - query: INSERT into health.eg_expense_lineitem (id, tenantid, billDetailId, headCode, amount, paidAmount, type, status, islineitempayable, createdby, createdTime, lastmodifiedby, lastModifiedTime, additionalDetails) values(?,?,?,?,?,?,?,?,false,?,?,?,?,?) ON CONFLICT (id,tenantid) DO UPDATE SET headcode=EXCLUDED.headcode, amount=EXCLUDED.amount, paidamount=EXCLUDED.paidamount, type=EXCLUDED.type, status=EXCLUDED.status, islineitempayable=EXCLUDED.islineitempayable, lastmodifiedby=EXCLUDED.lastmodifiedby, lastmodifiedtime=EXCLUDED.lastmodifiedtime, additionaldetails=EXCLUDED.additionaldetails;
+      basePath: $.bill.billDetails.*.lineItems.*
+      jsonMaps:
+
+       - jsonPath: $.bill.billDetails.*.lineItems.*.id
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)